src/components/tabs/panels.tsx

```diff
@@ -0,0 +1,104 @@
+/*
```

Comment:
Tabs panels container and panels. These are polymorphic components, which means that, by default, they are divs, but they can be other types of components, and Typescript will infer the correct acceptable attributes.